activesupport/lib/active_support/railtie.rb

```diff
@@ -49,17 +49,6 @@ class Railtie < Rails::Railtie # :nodoc:
       Date.beginning_of_week_default = beginning_of_week_default
     end
 
-    initializer "active_support.require_master_key" do |app|
-      if app.config.respond_to?(:require_master_key) && app.config.require_master_key
-        begin
-          app.credentials.key
-        rescue ActiveSupport::EncryptedFile::MissingKeyError => error
-          $stderr.puts error.message
-          exit 1
-        end
-      end
-    end
```

Comment:
How will the error look when booting a Rails app in production with `require_master_key = true`?

I added the rescue to prevent the backtrace from distracting from what were trying to communicate.

Reply:
If it remains as it is, the back trace will be showed. That is certainly noisy. 
I restored it.